static/styles/left-sidebar.scss

```diff
@@ -84,9 +85,9 @@ li.show-more-topics {
 
 #stream_filters {
     overflow: visible;
-    /* The -1px here prevents the scrollbar from going above the top of the container */
-    margin-top: -1px;
+    margin-top: 1px;
```

Comment:
What is this for? (It’s not on, e.g., `#user_presences`.)

Reply:
The last 2 screenshots refer to this, it's a very subtle change
If you see closely the top-left border of the selected stream one kinda seems sharp or broken
This probably because the stream_filters is getting underneath the `STREAMS` header
I fixed that adjusting the margin

![Screenshot from 2020-06-14 06-54-49](https://user-images.githubusercontent.com/32801674/84582687-f1ee7a80-ae0c-11ea-9c8e-db519dd9acc4.png)
![Screenshot from 2020-06-14 06-55-11](https://user-images.githubusercontent.com/32801674/84582688-f31fa780-ae0c-11ea-943c-89879f2a727b.png)

(Just realised that the commit message written for this is very poor, will fix that)

Reply:
No I understand why you’re taking out the `-1px` margin, but why are you adding in a `1px` margin?

Reply:
Ahh..my bad, that was not required